### DISCUSSION TITLE
Including Weblate as a part of an app/library name

### DISCUSSION THREAD

**Discussion by javierllorente** [WeblateOrg/weblate#7454] [Q&A] [answered]
Hello,

I was wondering about the naming policy of Weblate. Is it acceptable to include Weblate as a part of a non official app/library name? i.e: WeblateFX.

Thank you,
Javier

**Comment by nijel** [comment, marked as answer]
We don't have a trademark policy yet, but such usage should be okay - using the name for application operating with Weblate.
